src/storage/context.tsx

```diff
@@ -0,0 +1,43 @@
+import { createContext, Dispatch, useState } from 'react';
+import { getPersistContext, setPersistContext } from 'storage/persist';
+
+export const initialState = { isAuth: false };
+
+type User = {
+  id: number;
+  email: string;
+  full_name: string;
+  admin: boolean;
+  photo_path: boolean;
+};
+
+export type Context = {
+  isAuth?: boolean;
+  user?: User;
+  access_token?: string;
+};
+
+const AppContext = createContext<{
```

Comment:
There are possibilities to have more contexts while adding more features to the app. It could be called UserContext, making it more specific and if you need to store other data, you can create a new one.